extensions/panache/hibernate-orm-panache/runtime/src/main/java/io/quarkus/hibernate/orm/panache/runtime/AdditionalJpaOperations.java

```diff
@@ -53,6 +53,16 @@
         return new CustomCountPanacheQuery(em, jpaQuery, findQuery, countQuery, params);
     }
 
+    /**
+     * Calls EntityManager.clear()
+     * 
+     * @param jpaOperations
+     */
+    public static void clear(AbstractJpaOperations<?> jpaOperations) {
```

Comment:
Do we need this as the same operation was added in `RepositorySupport`?

Reply:
We only need one. I was not sure which one to prefer. But I'll remove this one then.

Reply:
I would say let's keep the `RepositorySupport` version as this will only be used in the Spring Data support.